### ISSUE TITLE
Only 7 Favorites radiostations

### ISSUE THREAD

**Issue by KolodyazhnyyI** [ChernyshovYuriy/OpenRadio#28]
Only 7 Favorites radio stations appear in favorites tab, when adding new radio stations to favorites-the oldest one disappear
Id like to ad like 12 radio stations...

**Comment by ChernyshovYuriy** [comment]
I have no issues with 12 or more stations. Does someone else experience the same as Igor?

**Comment by ChernyshovYuriy** [comment]
Do you still experience the issue?
Try to Upload and then Download on the same device after Clear Data, then use Send Logs, I might be able to spot something.
When Send Logs drop here a message with exact date, time and device model, so I can query from the database the logs.

**Comment by KolodyazhnyyI** [comment]
Yes,the problem persists, BUT when I add a new favorite station BY HAND,not
via search. it overwrites previous but still there are 7 stations in
favorites
But only if I add manually new station stream, ticking add to favorites

If I add from the search list- everything seems ok

OnePlus gm1900 22:57 30.10.23
